src/app/server-cluster/DefaultServerCluster.h

```diff
@@ -73,9 +81,20 @@ class DefaultServerCluster : public ServerClusterInterface
     /// Default implementation is a NOOP (no list items generated)
     CHIP_ERROR GeneratedCommands(const ConcreteClusterPath & path, DataModel::ListBuilder<CommandId> & builder) override;
 
+    /// Returns all global attributes that the spec defines in `7.13 Global Elements / Table 93: Global Attributes`
+    static Span<const DataModel::AttributeEntry> GlobalAttributes();
+
 protected:
+    ServerClusterContext * mContext = nullptr;
+
     void IncreaseDataVersion() { mDataVersion++; }
 
+    /// Marks that a specific attribute has changed value
+    ///
+    /// This increases cluster data version and if a cluster context is available it will
+    /// notify that the attribute has changed.
+    void NotifyAttributeChanged(AttributeId attributeId);
```

Comment:
This is one of the few places where the API loses the endpoint ID, which means 1 separate cluster instance is needed per endpoint (for example). Suggest using ConcreteAttributePath.

Reply:
Had separate discussion, in the end we figured
 https://github.com/project-chip/connectedhomeip/issues/37928 - asks for multiple paths support in the generic interface
 
 In this particular instance, the `DefaultServerCluster` is intended to be the typical/simple "one cluster that is stand-alone" so having a NotifyAttributeChanged seems to make sense.